.circleci/config.yml

```diff
@@ -39,6 +39,30 @@ jobs:
           key: user-dependency-cache-{{checksum "../user/src/package.json"}}
           paths:
             - ../user/src/node_modules
+      - run:
+          name: Replace Survey Project Creation Service environment variables
+          command: |
+            echo "CLOUD_STORAGE_PROVIDER=$CLOUD_STORAGE_PROVIDER" > ./src/.env
+            echo "CLOUD_STORAGE_ACCOUNTNAME=$CLOUD_STORAGE_ACCOUNTNAME" >> ./src/.env
+            echo "CLOUD_STORAGE_SECRET=$CLOUD_STORAGE_SECRET" >> ./src/.env
+            echo "CLOUD_STORAGE_BUCKETNAME=$CLOUD_STORAGE_BUCKETNAME" >> ./src/.env
+            echo "CLOUD_STORAGE_REGION=$CLOUD_STORAGE_REGION" >> ./src/.env
+            echo "CLOUD_ENDPOINT=$CLOUD_ENDPOINT" >> ./src/.env
+            ls -l ./src/.env # Check that the file is created
+            cat ./src/.env # Verify content for debugging
+
+
+      - run:
+          name: Replace User Service environment variables
```

Comment:
Do we have to do this? As we are not using any cloud related services of user service? @priyanka-TL